tools/scripts/start-local-registry.ts

```diff
@@ -58,7 +58,7 @@ function startLocalRegistry({
         ...`run ${localRegistryTarget} --location none --clear true`.split(' '),
         ...(storage ? [`--storage`, storage] : []),
       ],
-      stdio: 'pipe',
+      //stdio: 'pipe',
```

Comment:
Why is this commented out?

Reply:
I broke it with a recent refactoring of `executeProcess`.